File: pset6/mario.py
import sys
import cs50

def main():
    
    while True:
        print("enter a non-negative number less than 24: ", end="")
        height = cs50.get_int()
        if height > 0 and height < 24:
            break
    
    for row in range(1, height + 1):
        spaces = height - (row)
        print(" " * spaces, end="")
        print("#" * (row), end="")
        print("  ", end="")
        print("#" * (row))
    
        
if __name__ == "__main__":
    main()
    
    

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    int height = 0;
    
    // prompt user for a height integer greater than 0 and less than 24
    do {
        printf("Enter a non-negative integer less than 24: ");
        height = get_int();
    }
    while (height < 0 || height > 23);

    for (int row = 1; row <= height; row++){
        
        // declare spaces variable
        int spaces = height - row;
        
        //print number of spaces
        for (int j = 0; j < spaces; j++){
            printf(" ");
        }
        
        // print left pyramid hashes
        for (int j = 0; j < row; j++){
            printf("#");
        }
        
        // print two spaces
        for (int j = 0; j < 2; j++){
            printf(" ");
        }
        
        // print right pyramid
        for (int j = 0; j < row; j++){
            printf("#");
        }
        
        // print newline
        printf("\n");
        
    }
    
}